app/controllers/users_controller.rb

```diff
@@ -0,0 +1,17 @@
+class UsersController < ApplicationController
+
+  def add_case
```

Comment:
(Let me know if I'm wrong and we're keeping track of call list for an individual user with this method! Or if we can adapt it to keep call list in session)